v2/googlecloud-to-elasticsearch/src/test/java/com/google/cloud/teleport/v2/elasticsearch/templates/PubSubToElasticsearchTest.java

```diff
@@ -179,7 +179,7 @@ public void testPubSubToElasticsearchUdfE2E() {
     PubSubToElasticsearchOptions options =
         TestPipeline.testingPipelineOptions().as(PubSubToElasticsearchOptions.class);
 
-    options.setErrorOutputTable("test:dataset.table");
+    options.setErrorOutputTopic("test:pubsub.topic");
```

Comment:
Pub/Sub topic should be specified in projects/\<project-id>/topics/\<topic-name> format.